tests/system/action/base.py

```diff
@@ -59,6 +59,14 @@ def request_json(self, payload: Payload, anonymous: bool = False) -> Response:
         client = self.client if not anonymous else self.anon_client
         return client.post(ACTION_URL, json=payload)
 
+    def request_lang(
+        self, action: str, data: Dict[str, Any], lang: str = "en_US"
+    ) -> Response:
+        payload = [{"action": action, "data": [data]}]
+        headers = {"Accept_Language": lang}
+
+        return self.client.post(ACTION_URL, json=payload, headers=headers)
```

Comment:
I would rather integrate the `lang` parameter into the other request function to have it available everywhere and not have to differentiate between translate request and other ones.

Reply:
Done.